### DISCUSSION TITLE
tutorials/getting-started-with-graphos/04-creating-a-supergraph-in-graphos

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#123] [course feedback]
# tutorials/getting-started-with-graphos/04-creating-a-supergraph-in-graphos

https://www.apollographql.com/tutorials/getting-started-with-graphos/04-creating-a-supergraph-in-graphos

**Comment by sunnytoolsmiths** [comment]
I am trying to connect the subgraph using rover. In the following command, what should I put in the schema parameter? I kept getting error messages about " Invalid path. No file found at ./schema.graphql"

Here is the command I used: 
APOLLO_KEY=service:blahblahblah \
  rover subgraph publish Observability-Demo-x8oh8n@current \
  --schema ./schema.graphql \
  --name recipes \
  --routing-url https://poetic-plates-recipes-api.herokuapp.com/

**Reply by lizhennessy** [reply]
Hi there @sunnytoolsmiths! It looks like you're trying to connect to the subgraph running remotely at `https://poetic-plates-recipes-api.herokuapp.com/`, is that correct? Publishing with Rover CLI in this way requires us to specify the path to our subgraph's schema, which is probably why you're seeing an error. (We generally use it when we're running our subgraph locally, and have the schema file with its changes that we want to publish to our graph)

A couple of questions: were you able to complete the steps described in this lesson to create your supergraph? This will take care of adding the subgraph at `https://poetic-plates-recipes-api.herokuapp.com/` to GraphOS for us to query in the next part. Or is there a different goal you have in mind, using Rover instead of the onboarding flow in Studio? I'm happy to help troubleshoot further, just let me know. :)

**Comment by sunnytoolsmiths** [comment]
Hey I am not seeing the same screen as presented here. When I logged into the studio, clicks on create a new graph, it will immediately provide the meta data about the super graph in the command and I won't be able to add ay remote endpoint directly.

**Comment by devniel93** [comment]
I tried to create an organization with a serverless plan but it is no longer allowed according to https://www.apollographql.com/docs/graphos/routing/cloud/serverless.
And to follow this tutorial I need that plan!!!!

**Reply by mabuyo** [reply]
Hi there @devniel93 - I'm sorry for the confusion. This course has actually been deprecated and it's marked as such at the top beside the course title. I recommend taking [Voyage I](https://www.apollographql.com/tutorials/voyage-part1) or [one of the "Federation with ..." courses](https://www.apollographql.com/tutorials/browse?categories=federation) that match the language you're most comfortable with to learn similar concepts covered in this course.
